socorro/external/boto/connection_context.py

```diff
@@ -87,7 +89,7 @@ def is_operational_exception(self, x):
             return True
         return False
 
-    def __init__(self, config, quit_check_callback=None):
+    def __init__(self, config, namespace='', quit_check_callback=None):
```

Comment:
Oops--I didn't catch that you changed the `__init__` for the base class. You're going to need to update all the subclasses and all the points that one of those classes gets instantiated, too.

Reply:
Does https://github.com/mozilla-services/socorro/pull/4493/commits/7d8752ef7a95f3d3faaaad61912d3f3e81413a90 make sense?

Reply:
This looks good, but it's missing something: we need to pass in the namespace when we create the connection context class:

https://github.com/mozilla-services/socorro/blob/a1100df887a077596539f1fc08c63754705ecd20/socorro/external/boto/crashstorage.py#L91

Because we have metrics set up to log metrics, you can see the keys and values getting spit out when running the processor. Right now it shows this:

```
processor_1      | 2018-06-27 13:33:26,276 INFO - markus - Thread-2 - METRICS|2018-06-27 13:33:26|histogram|submit|70.259809494|#kind:processed_crash,outcome:successful
processor_1      | 2018-06-27 13:33:26,276 INFO - markus - Thread-2 - METRICS|2018-06-27 13:33:26|timing|processor.s3.BotoS3CrashStorage.save_raw_and_processed|77.6069164276|
processor_1      | 2018-06-27 13:33:26,323 INFO - markus - Thread-2 - METRICS|2018-06-27 13:33:26|histogram|processor.elasticsearch.raw_crash_size|14029|
processor_1      | 2018-06-27 13:33:26,324 INFO - markus - Thread-2 - METRICS|2018-06-27 13:33:26|histogram|processor.elasticsearch.processed_crash_size|132336|
processor_1      | 2018-06-27 13:33:26,756 INFO - processor - Thread-2 - Created new elasticsearch index: socorro201826
processor_1      | 2018-06-27 13:33:27,177 INFO - markus - Thread-2 - METRICS|2018-06-27 13:33:27|histogram|processor.elasticsearch.index|420.194149017|#outcome:successful
processor_1      | 2018-06-27 13:33:27,177 INFO - markus - Thread-2 - METRICS|2018-06-27 13:33:27|timing|processor.es.ESCrashStorageRedactedJsonDump.save_raw_and_processed|854.610919952|
processor_1      | 2018-06-27 13:33:27,177 INFO - markus - Thread-2 - METRICS|2018-06-27 13:33:27|incr|processor.save_raw_and_processed|1|
processor_1      | 2018-06-27 13:33:27,280 INFO - markus - Thread-2 - METRICS|2018-06-27 13:33:27|histogram|submit|101.981878281|#kind:crash_report,outcome:successful
processor_1      | 2018-06-27 13:33:27,281 INFO - markus - Thread-2 - METRICS|2018-06-27 13:33:27|timing|processor.telemetry.TelemetryBotoS3CrashStorage.save_raw_and_processed|103.121042252|
```

The `submit` key is missing the namespace.